### PR TITLE
Page question result

### PR DESCRIPTION
**Pull request by anshulraheja**
1. Question Page
2. Result Page

### CODE REVIEW

File: quiz-app/css/question.css [hunk 1]
```diff
@@ -0,0 +1,150 @@
+.question-container{
+    display: flex;
+    align-items: center;
+    justify-content:center;
```
Comment (ShekharDhangar) [inline quiz-app/css/question.css:4]: use flex as helper class and include all properties in it

File: quiz-app/css/question.css [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+}
+
+.quiz-box header{
+    height: 70px;
+    padding: 0 30px;
+    background: var(--COLOR-WHITE);
+    border-radius: 5px 5px 0 0;
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
```
Comment (ShekharDhangar) [inline quiz-app/css/question.css:20]: same use flex as helper class and overwrite specific changes in the class of element you are applying the properties

File: quiz-app/css/question.css [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+
+.quiz-box-timer .timer_sec{
+    font-size: 18px;
+    font-weight: 500;
+    height: 30px;
+    width: 45px;
+    color: var(--COLOR-WHITE);
+    border-radius: 5px;
+    line-height: 30px;
+    text-align: center;
```
Comment (ShekharDhangar) [inline quiz-app/css/question.css:55]: use text-center as helper class, in this way you don't have to repeat the code your self

File: quiz-app/css/result.css [hunk 4]
```diff
@@ -0,0 +1,56 @@
+.result-container{
+    display: flex;
+    justify-content: center;
+    align-items: center;
```
Comment (ShekharDhangar) [inline quiz-app/css/result.css:4]: You have used flex a lot and repeated the properties in every class, so create a helper class as said.

File: quiz-app/css/result.css [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+    color: var(--COLOR-BLACK);
+    text-align: center;
+}
+.result_box-btn{
+    display: flex;
+    margin: 1.25rem 0;
+}
+.result_box-btn-replay, .result_box-btn-quit{
+    margin: 0 0.6rem;
+    padding: 1rem;
```
Comment (ShekharDhangar) [inline quiz-app/css/result.css:34]: use padding,margin as helper class

**Review by ShekharDhangar — CHANGES_REQUESTED** [review]
Please do the suggested changes

**Review by VLeads — APPROVED** [review]
Code is clean. Good to go !!

**Pull request merged**